boards/arm/v2m_beetle/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
-if(CONFIG_PINMUX_BEETLE)
-  zephyr_library()
-  zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
-  zephyr_library_sources(pinmux.c)
-endif()
+zephyr_library()
+zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
+zephyr_library_sources(pinmux.c)
```

Comment:
I see this board is including drivers/pinmux.h for nothing, can be removed as well.

Reply:
you are right, good point, all of them in fact, and other boards too... I'll have a wider look and clean those up as well